Client/Frontend/Settings/PasswordDetailViewController.swift

```diff
@@ -129,10 +129,10 @@ extension PasswordDetailViewController: UITableViewDataSource {
             breachDetailView.setup(breach)
             breachDetailView.applyTheme(theme: themeManager.currentTheme)
 
-            breachDetailView.learnMoreButton.addTarget(self, action: #selector(PasswordDetailViewController.didTapBreachLearnMore), for: .touchUpInside)
+            breachDetailView.viewBreachDetailsButton.addTarget(self, action: #selector(PasswordDetailViewController.didTapBreachLearnMore), for: .touchUpInside)
```

Comment:
Here is my proposed solution for this problem, to make sure the properties remain private, they belong to `BreachAlertsDetailView`: 
1. We add a closure in `BreachAlertsDetailView`:
`var onTapLearnMore: (() -> Void)?`
2. We create a `#selector` method in `BreachAlertsDetailView` and we call there our previously declared closure.
 `@objc
    private func didTapBreachLearnMore() {
        onTapLearnMore?()
    }`
3. We add the target to `learnMoreButton` in `BreachAlertsDetailView`
`button.addTarget(self, action: #selector(self.didTapBreachLearnMore), for: .touchUpInside)`
4. We can now remove the `breachDetailView.viewBreachDetailsButton.addTarget(self, action: #selector(PasswordDetailViewController.didTapBreachLearnMore), for: .touchUpInside)` from `PasswordDetailViewController` and use our closure to send the action further:
`breachDetailView.onTapLearnMore = { self.didTapBreachLearnMore() }`

- We can repeat the same process for the other property.